fn/client/src/reducers/Cart-reducer.js

```diff
@@ -5,31 +5,31 @@ const initialState = { cartNumbers: 0, cartProducts: [] }
 const userId = JSON.parse(localStorage.getItem("userId"));
 const productCollection = JSON.parse(localStorage.getItem("products-collection"));
 const localCartNumbers = JSON.parse(localStorage.getItem("cart-numbers"));
+const accessToken = JSON.parse(localStorage.getItem('accessToken'))
 
-const saveCart = async (userId, cart) => {
-  axios.put(`https://lv487node-backend.herokuapp.com/users/cart/${userId}`, { cart });
+const saveCart = async (userId, data, token) => {
+  axios({ method: 'PUT', url: `http://localhost:5000/users/cart/${userId}`, data, headers: { "x-auth-token": token } });
 }
-
-const setInitial = async () => {
-    if (userId) {
-      const res = await axios.get(`https://lv487node-backend.herokuapp.com/users/${userId}`);
-      const { cart } = res.data.user
-      if (!cart) {
-        saveCart(userId, { cartNumbers: 0, cartProducts: [] })
-        return
-      }
-      initialState.cartNumbers = cart.cartNumbers
-      initialState.cartProducts = cart.cartProducts
+const setInitial = async (token) => {
+  if (userId) {
+    const res = await axios({ method: 'GET', url: `http://localhost:5000/users/${userId}`, headers: { "x-auth-token": token } })
```

Comment:
same about basicUrl it should be in config

Reply:
fixed